boxoffice/views/admin_discount.py

```diff
@@ -51,11 +65,35 @@ def jsonify_discount_policies(data_dict):
 @render_with({'text/html': 'index.html', 'application/json': jsonify_discount_policies}, json=True)
 def admin_discount_policies(organization):
     query = request.args.get('search')
-    if query:
-        discount_policies = DiscountPolicy.query.filter(DiscountPolicy.title.ilike('%{query}%'.format(query=query))).all()
+    page = request.args.get('page', 1)
+
+    if request.is_xhr:
```

Comment:
@shreyas-satish the actual data is loaded when the ajax call is made. when you load the page, it loads the JS that calls the endpoint again to load data. so if we dont have this here, the same db queries will be called twice but only the ajax call will update the page.